sormas-api/src/main/resources/strings.properties

```diff
@@ -991,7 +1000,10 @@ infoNoAccessToPersonEntities = You don't have the necessary user rights to view
 infoDashboardFinalLaboratoryResult = When a case has multiple samples, only the final laboratory result of the sample with the latest sample collection date is considered.
 infoBulkProcess = %d selected entries are currently being processed. This may take a while. The progress will be updated for each 20 entries that have been processed.
 infoBulkProcessFinished = <b>All selected entries have been processed!</b><br/>You can now close this window.
-infoBulkProcessFinishedWithSkips = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction or already archived.
+infoBulkProcessFinishedWithIneligibleItems = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, because they were not eligible.
+infoBulkDeleteProcessFinishedWithSkips = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction.
```

Comment:
What's the purpose of this string? It's called "...DeleteProcess...", but there's no reference to deletion in the string; instead, it's basically the next string minus the eligible keyword.

Reply:
infoBulkProcessFinishedWithIneligibleItems will appear for all the bulk processes which will be ineligible ( as I've mentioned in one comment from above ex: for now the archive without archive rights (till this will not be solved by hiding the button, the delete of events with event participants and any other scenario which I still have to check for other bulk processes). It will be a general message for all bulk processes. 
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/02a6e9a5-2078-405f-aa05-4ccb8b12c172)

infoBulkDeleteProcessFinishedWithSkips = will be used only for delete because in a call @adinaflorea9 observed that the message contains also the archive and for delete is not correct to displayed that the reason for unprocessed entities is "outside your jurisdiction or already archived" because archived entities can also be deleted so for the delete I've used a message without the "archiving" part and we also know that is not the "not eligible" problem because I've handled in other condition that one.
This specific message ( only for delete) will be passed to the DeleteRestoreController ( for soft delete) and PermanentDeleteController ( for permanent delete) 

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/586ad179-84e3-402e-8b0f-5321454d5501)

For all the other bulk processes, we will use the general message:
infoBulkProcessFinishedWithSkips = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction, already archived or not eligible.

For Edit:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/9d6444bf-70e2-41ea-9571-3bc19ebaf979)

For Archive/Dearchive:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/c57f5d1f-ae66-4815-ab54-ebad8393909f)

Should I change it? If yes, how would you want to be changed?

Reply:
I think I would just call the property infoBulkProcessFinishedWithSkips, simply because there's no deletion reference in its text, even if we (right now) exclusively use it in the deletion process

Reply:
It's not visible from the above PR section, but we already have the infoBulkProcessFinishedWithSkips.
This will be used for other bulk processes ( not delete) because contains the "archive part" which would be not true for deletion.

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/d5331466-2a86-43e5-b251-6ad39061b9c4)

So right now we have the following:

- **infoBulkProcessFinishedWithSkips** = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction, already archived or not eligible.
- **infoBulkDeleteProcessFinishedWithSkips** = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction.

How would you want to rename the infoBulkDeleteProcessFinishedWithSkips? (since infoBulkProcessFinishedWithSkips is already taken :D)

Reply:
I'd go with `infoBulkProcessFinishedWithSkipsOutsideJurisdiction`